### ISSUE TITLE
TilingSprite not working with PIXI v5

### ISSUE THREAD

**Issue by hoeugene** [michalochman/react-pixi-fiber#118]
### Description
"TypeError: Cannot read property 'TilingSprite' of undefined" when using TilingSprite.
This should be caused by TilingSprite rendering from PIXI.extras.TilingSprite, which has been deprecated in [v5 of pixi](http://pixijs.download/dev/docs/PIXI.extras.TilingSprite.html). I've been able to fix this by rendering from PIXI.TilingSprite

### Additional info

- react-pixi-fiber version: 0.8.1
- React version: 16.8.6
- ReactDOM version: 16.8.6
- PixiJS version: 5.0.4

**Comment by michalochman** [comment]
Hi @hoeugene, I would suggest creating [Custom Component](https://github.com/michalochman/react-pixi-fiber#custom-components) which creates an instance of `PIXI.TilingSprite`, something like:

```jsx
const TilingSprite = CustomPIXIComponent({
  customDisplayObject: props => new PIXI.TilingSprite(props),
}, "TilingSprite");
```

I will need to address PixiJS v4 and v5 compatibility some way.

**Comment by z-ax** [comment]
Yeah, the same with `ParticleContainer`.

**Comment by michalochman** [comment]
@hoeugene @z-ax this should be fixed in next version – v0.9.0
